### PR TITLE
Use official product name "Spring Data for ScalarDB" in the sample applications

### PR DESCRIPTION
**Pull request by komamitsu**
We decided the official product name of the integration of Spring Data and ScalarDB. This PR uses the official name  "Spring Data JDBC for ScalarDB" in the 2 sample applications.

### CODE REVIEW

File: spring-data-multi-storage-transaction-sample/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Spring Data Integration Sample with Multi-storage Transactions
+# Sample application of Spring Data JDBC for ScalarDB with Multi-storage Transactions
```
Comment (komamitsu) [inline spring-data-multi-storage-transaction-sample/README.md:1]: I thought "Spring Data JDBC sample for ScalarDB" or "Spring Data JDBC for ScalarDB sample" was a bit confusing. So I started with "Sample application of" to make the title clearer.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by josh-wong — APPROVED** [review]
LGTM!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you! 🙇

**Pull request merged**